k8s_itests/deployments/cluster.yaml

```diff
@@ -0,0 +1,22 @@
+---
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+network: k8s_kind
```

Comment:
this is not going to work in travis without more shenanigans, pre-process it to be absent by default and `k8s_kind` in yelp envs. the attribute is only supported by our fork of kind.